src/services/file-storages/aws/aws-service.ts

```
@@ -76,19 +69,19 @@ export class AWSService implements FileStorage {
         },
         tags: [],
         queueSize: 4,
-        partSize: 1024 * 1024 * 5,
+        partSize: 1024 * 1024 * 5, // Maximum part size is 5MB
         leavePartsOnError: false, // optional manually handle dropped parts
       });
 
       const resDone = await res.done();
 
       if (!resDone) {
-        throw new AWSS3UploadError(dictionary.errors.awsUploadIssue);
+        throw new Error(dictionary.errors.awsUploadIssue);
       }
 
-      return `https://${this.awsS3Bucket}.s3.amazonaws.com/${fileName}`;
+      return resDone.Location || '';
```

Why the or statement?

---

if location is undefined then we should throw an Error